### ISSUE TITLE
importYaml is not compatible with MATLAB 2022b 

### ISSUE THREAD

**Issue by feiranl** [SysBioChalmers/Human-GEM#614]
#### Reproducing these results:

1. In this environment IOS system, MATLAB2022b with Human-GEM main branch and RAVEN 2.8.1
2. Run model = importYaml('Human-GEM.yml');
3. See error:

```
Error using readlines
Argument must be a text scalar.

Error in importYaml (line 38)
    line_raw=readlines(yamlFilename');
```
I checked the yamlFilename =  'Human-GEM.yml'
yamlFilename' =

  13×1 char array

    'H'
    'u'
    'm'
    'a'
    'n'
    '-'
    'G'
    'E'
    'M'
    '.'
    'y'
    'm'
    'l'

After removing the transpose sign, the code works fine.

I double checked the corresponding code `readYAMLmodel` in RAVEN using
model = readYAMLmodel('Human-GEM.yml'); which works fine, and the corresponding line is     line_raw=readlines(fileName); which doesn't contain the transpose sign.

According to the blame file, @edkerk coded for this line in Human-GEM before. Hi Ed, do you have any suggestion for this one? https://github.com/SysBioChalmers/Human-GEM/commit/9aea8ae03e5dcd3e3623cb745510e43aa02bd3fb

**Comment by feiranl** [comment]
Thanks for the quick response! 

I also vote for having a model read function in this repo to keep it simple for now.

**Comment by feiranl** [comment]
Fixed in #619
